workspaces/scaffolder-backend-module-servicenow/plugins/scaffolder-backend-module-servicenow/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "@backstage-community/plugin-scaffolder-backend-module-servicenow",
+  "description": "The servicenow custom actions",
+  "version": "2.2.0",
```

Comment:
note that the latest release of the janus version is 2.2.1 https://www.npmjs.com/package/@janus-idp/backstage-scaffolder-backend-module-servicenow?activeTab=versions